gcp/modules/couchdb/values.yaml

```diff
@@ -40,6 +40,8 @@ couchdbConfig:
     bind_address: any
     # http://docs.couchdb.org/en/stable/config/auth.html#authentication-configuration
     require_valid_user: true
+  httpd:
+    WWW-Authenticate: "Basic realm=\"CouchDB\""
```

Comment:
Security requirement from GPII-3341 says: "HTTP endpoint should not be exposed and should be disabled". Set this only for dev environments?

Reply:
This does not expose any new port, nor endpoints. It's still very same port (`5984`) and endpoints that are currently used by apps and are accessible by direct http call when you supply the credentials, the only difference is that this setting will trigger auth popup when accessed in the browser.